projects/media-viewer/src/assets/sass/bookmarks.scss

```diff
@@ -0,0 +1,47 @@
+mv-side-bar {
```

Comment:
it would be good to have some visual indication that you're hovering over the buttons, so that it's clear you can click them

Reply:
Ok I will add that now!